<?xml version="1.0" encoding="UTF-8"?><record_update table="sys_ui_message">
    <sys_ui_message action="INSERT_OR_UPDATE">
        <application/>
        <code/>
        <commented_message/>
        <key>Scope validation attempt "{0}" of "{1}" failed. Scope is not 'Global', not found on parent instance and lookup in app store failed due to comm. error. Make sure app store is available. See parent instance log for more info. Will re-try in "{2}"</key>
        <language>uk</language>
        <message>Невдала спроба перевірити область дії ("{0}" із "{1}"). Область дії не є глобальною. Вона не знайдена в батьківському примірнику, а пошук у магазині програм не вдався через помилку в команді. Переконайтеся, що магазин програм доступний. Додаткові відомості див. у журналі батьківського примірника. Повторна спроба буде виконана через "{2}"</message>
        <sys_class_name>sys_ui_message</sys_class_name>
        <sys_created_by>bogdan.felinskyi</sys_created_by>
        <sys_created_on>2022-03-23 17:40:05</sys_created_on>
        <sys_id>b16fb82b1b96c910c069657bab4bcb64</sys_id>
        <sys_mod_count>0</sys_mod_count>
        <sys_name>Scope validation attempt "{0}" of "{1}" failed. Scope is not 'Global', not found on parent instance and lookup in app store failed due to comm. error. Make sure app store is available. See parent instance log for more info. Will re-try in "{2}"</sys_name>
        <sys_package display_value="I18N: Ukrainian Translations" source="176ebda287b38910f4b131570cbb3551">176ebda287b38910f4b131570cbb3551</sys_package>
        <sys_policy/>
        <sys_scope display_value="I18N: Ukrainian Translations">176ebda287b38910f4b131570cbb3551</sys_scope>
        <sys_update_name>sys_ui_message_b16fb82b1b96c910c069657bab4bcb64</sys_update_name>
        <sys_updated_by>bogdan.felinskyi</sys_updated_by>
        <sys_updated_on>2022-03-23 17:40:05</sys_updated_on>
    </sys_ui_message>
</record_update>
